### PR TITLE
Bugfix/temporary table

### PR DESCRIPTION
**Pull request by bwalsh**
This PR:
* addresses bug where sql error is created if input record count > 1000

### CODE REVIEW

**Review by matthewpeterkort — APPROVED** [review]
The table fix worked for me. Was able to import >1000 observations into elastic through the gen3_util.

**Review by lbeckman314 — COMMENTED** [review]
<!-->

**Review by lbeckman314 — APPROVED** [review]
LGTM! Trying out now in local ETL pod but to try out new commands.

---

**Successful install on local machine:**
```sh
(venv) ➜  submission git:(bugfix/temporary_table) python setup.py install
# running install
# ...
# Finished processing dependencies for aced-submission==0.0.8rc1
```

**General help command:**
```sh
(venv) ➜  submission git:(bugfix/temporary_table) aced_submission --help
Usage: aced_submission [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  meta            Project data (ResearchStudy, ResearchSubjects, Patient,...
  files           Project files (Omics, Imaging, ...).
  fhir            Adds, retrieves, and deletes FHIR resources in a FHIR...
  schema-publish  Copy dictionary to s3 (note:aws cli dependency)
```

**Meta help command (showing `counts` subcommand):**
```sh
(venv) ➜  submission git:(bugfix/temporary_table) aced_submission meta flat --help
Usage: aced_submission meta flat [OPTIONS] COMMAND [ARGS]...

  Load flat indexes into elasticsearch.

Options:
  --help  Show this message and exit.

Commands:
  counts               Count the number of patients, observations, and...
  denormalize-patient  Gather Patient, FamilyHistory, Condition into sqlite...
  load                 Gen3 Elastic Search data into guppy (patient,...
  rm                   Delete items from elastic index for project_id.
```

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
     # Versions should comply with PEP 440:
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='0.0.7',  # Required
+    version='0.0.8rc1',  # Required
```
Comment (matthewpeterkort) [inline setup.py]: this should be 0.0.8rc2 to match 
https://pypi.org/project/aced-submission/0.0.8rc2/ and 
https://github.com/ACED-IDP/aced_etl_pod/blob/8dc8511749b55ced07e1edd92149410d229f0f7f/etl-job/Dockerfile#L48

Comment (bwalsh) [reply setup.py]: done. thanks.

**Pull request merged**